data/strings/types_strings.txt

```diff
@@ -27200,41 +26967,6 @@
     tr = Kızak Pisti
     uk = Траса для санок
 
-  [type.building_part]
```

Comment:
Are you sure that it should be deleted? I assume that some buildings may not show "Building" on their selections. Should it be replaced to the `ref = building` ?

@vng

Reply:
i moved it to be with the other type.building translations (line 5301) and made it refer to type.building